src/metabase/models.py

```diff
@@ -151,4 +150,217 @@ class UsageStats(BaseModel):
+  parent_id: int
+  id: int
+  archived_directly: str
+  entity_id: str
+  location: str
+  namespace: str
+  is_personal: bool
+  created_at: str
+  can_delete: bool 
+  effective_ancestors: list['Collection']
+
+class SearchResults(BaseModel):
+
+  description: str
+  archived: bool
+  collection_position: int
+  table_id: int
+  bookmark: bool
+  can_write: bool
+  database_id: int
+  last_editor_common_name: str
+  database_name: str
+  pk_ref: str
+  name: str
+  type: str
+  table_schema: str
+  creator_common_name: str
+  collection_authority_level: str
+  creator_id: str
+  updated_at: str
+  moderated_status: str
+  model_index_id: str
+  model_id: str
+  effective_location: str
+  last_editor_id: str
+  model_name: str
+  id: int
+  last_edited_at: str
+  table_description: str
+  dashboardcard_count: int
+  context: str
+  display: str
+  scores: list['Scores']
+  description: str
+  initial_sync_status: str
+  table_name: str
+  location: str
+  collection: dict
+  created_at: str
+  model: str
+
+class Scores(BaseModel):
+    weight: float
+    score: float
+    name: str
+    column: str
+    match: str
+
+class TableMetaData(BaseModel):
+  description: str | None = None
+  entity_type: str | None = None
+  view_count: int
+  _schema: str | None = None
+  database_require_filter: str | None = None
+  db: 'TableDB'
+  show_in_getting_started: bool
+  name: str | None = None
+  fields: List['TableFields']
+  caveats: str | None = None
+  segments: List[str]
+  dimension_options: Union[List, Dict, None] = None
+  updated_at: str | None = None
+  active: bool
+  id: int
+  db_id: int
+  entity_id: str | None = None
+  visibility_type: str | None = None
+  field_order: str | None = None
+  is_upload: bool
+  initial_sync_status: str | None = None
+  display_name: str | None = None
+  metrics: List
+  created_at: str | None = None
+  estimated_row_count: str | None = None
+  points_of_interest: str | None = None
+
+class TableDB(BaseModel):
+    uploads_schema_name: str | None = None
+    description: str | None = None
+    features: List
+    uploads_table_prefix: str | None = None
+    cache_field_values_schedule: str | None = None
+    timezone: str | None = None
+    is_attached_dwh: bool
+    auto_run_queries: bool
+    metadata_sync_schedule: str | None = None
+    name: str | None = None
+    settings: str | None = None
+    caveats: str | None = None
+    creator_id: int
+    is_full_sync: bool
+    updated_at: str | None = None
+    cache_ttl: str | None = None
+    details: str | None = None
+    is_sample: bool
+    id: int
+    is_on_demand: bool
+    entity_id: str | None = None
+    engine: str | None = None
+    initial_sync_status: str | None = None
+    is_audit: bool
+    dbms_version: Dict[str, Any]
+    uploads_enabled: bool
+    refingerprint: str | None = None
+    created_at: str | None = None
+    points_of_interest: str | None = None
+
+class TableFields(BaseModel):
+      description: str | None = None
```

Comment:
Are these indented properly? I think they need to be moved over. There are a couple placesin this file like that